training/src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,103 @@
+package wolox.training.models;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+
+@Entity
+@JsonIdentityInfo(
+    generator = ObjectIdGenerators.PropertyGenerator.class,
+    property = "id")
+public class Users {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.MERGE)
+    @JoinTable(name = "user_book",
+        inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"),
+        joinColumns = @JoinColumn(name = "book_id",
+            referencedColumnName = "id"))
```

Comment:
If you don't use the `@JoinTable` annotation, spring will automatically create a table called `users_books` and its columns will be called `books_id` and `users_id`. If you **really** want the names you specified, then leave this annotation. In my opinion, I don't think you need it: it's not a business requirement and why bother in doing something that can be done by a tool?
We only use `@JoinTable` when we need to comply with a naming convention that does not match the one provided by springboot.